views/includes/check-your-answers/update/managing-officers-addresses.html

```diff
@@ -1,9 +1,11 @@
 {% if managing_officer_individual %}
-    {% set residentialOrPrincipalAddressText = "Home address" %}
-    {% set sameAddressText = "The correspondence address is the same as the home address" %}
+  {% set residentialOrPrincipalAddressText = "Home address" %}
+  {% set sameAddressText = "The correspondence address is the same as the home address" %}
+  {% set changeResidentialOrPrincipalAddressHtml = OE_CONFIGS.CHANGE_RESIDENTIAL_ADDRESS %}
```

Comment:
this is not "Html" - just a link so name seems somewhat wrong.

Reply:
fixed